src/components/PeopleFilters.tsx

```diff
@@ -33,63 +59,45 @@
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {['16', '17', '18', '19', '20'].map(Century => (
+              <SearchLink
+                data-cy="century"
+                className={cn('button mr-1', {
+                  'is-info': Century.includes(Century),
+                })}
+                params={{
+                  century: Century.includes(Century)
+                    ? centuries.filter(curr => curr !== Century)
+                    : [...centuries, Century],
```

Comment:
The logic for toggling centuries in the `params` object seems incorrect. Ensure that `centuries` is being checked correctly. The condition should toggle the presence of `Century` in the `centuries` array.